tools/lib/capitalization.py

```diff
@@ -138,6 +138,8 @@
     r"your-organization",
     # Used in invite modal
     r"or",
+    r"Welcome Bot",
+    r"none",
```

Comment:
Can you post a screenshot of where this appears tagged for translation in the UI?